pkg/scheduler/scheduler.go

```diff
@@ -50,6 +53,7 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/metrics"
 	"k8s.io/kubernetes/pkg/scheduler/profile"
 	"k8s.io/kubernetes/pkg/scheduler/util"
+	utiltrace "k8s.io/utils/trace"
```

Comment:
nit: it seems `trace` is enough.

Reply:
Thanks !
But I notice that the `"k8s.io/utils/trace"` are imported as `utiltrace` for most situations. So I prefer to keep it.